python_examples/sim_speeds/Sim_Speeds_plot.txt

```diff
@@ -0,0 +1,13 @@
+basic       1000        60      1       2   Y
```

Comment:
Again if this is going on main we want it to be tidy - happy for you to keep it like this in a separate branch but not on main. I'm not even sure if we need this file, it might be nicer to store it in a readable way (as in the previous txt file) and I can help you write a function to convert "min:secs" into total seconds